app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return
```

Comment:
it is better to raise informative error so that it is clear to the user if something goes wrong